packages/devtools_app/lib/src/shared/table/table_data.dart

```diff
@@ -79,6 +79,8 @@ abstract class ColumnData<T> {
   /// Get the cell's display value from the given [dataObject].
   String getDisplayValue(T dataObject) =>
       getValue(dataObject)?.toString() ?? '';
+  
+  Widget? getDisplayWidget(T dataObject, BuildContext context) => null ;
```

Comment:
Delete this and have your columns implement `ColumnRenderer` instead. Example: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/screens/logging/_kind_column.dart#L13